.gitignore

```diff
@@ -1,5 +1,5 @@
-lib
```

Comment:
Can't we just revert this file to origin/master?

Reply:
Agreed with tad; this doesn't seem to be something we'd want to commit.